docs/src/SUMMARY.md

```diff
@@ -0,0 +1,31 @@
+# Summary
+- Getting started
+  - [Introduction](./intro.md)
+  - [Prerequisites](./prerequisites.md)
+  - [Provider configuration](./provider.md)
+  - [Quickstart](./deploy.md)
+- User Guide
+  - [Topology](./guide/topology.md)
+  - [Network](./guide/network.md)
+  - [Bastion](./guide/bastion.md)
+  - [Cluster](./guide/cluster.md)
+  - [Operator](./guide/operator.md)
+  - [Workers](./guide/workers.md)
+  - [Utilities](./guide/utilities.md)
+  - [Extensions](./guide/extensions.md)
+  - [Cloud init](./guide/cloud_init.md)
+  - [Load balancers](./guide/loadbalancers.md)
+  - [Resource Manager](./guide/rms.md)
+  - [Upgrading](./guide/upgrade.md)
+- Reference
+  - Inputs
```

Comment:
I'm wondering whether we can document this as an API reference instead.

Reply:
WIP/already changed